src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -74,18 +75,24 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
 
   const { ApplicationView }: PluginComponents = pluginProvider.getPluginElement(pluginCode)
 
+  const PluginComponent = (
+    <ApplicationView
+      onUpdate={onUpdate}
+      onSave={onSave}
+      validationState={validationState || { isValid: true }}
+      // TO-DO: ensure validationState gets calculated BEFORE rendering this child, so we don't need this fallback.
+      {...props}
+    />
+  )
+
   return (
     <ErrorBoundary pluginCode={pluginCode}>
       <React.Suspense fallback="Loading Plugin">
-        {
-          <ApplicationView
-            onUpdate={onUpdate}
-            onSave={onSave}
-            validationState={validationState || { isValid: true }}
-            // TO-DO: ensure validationState gets calculated BEFORE rendering this child, so we don't need this fallback.
-            {...props}
-          />
-        }
+        {isRequired ? (
+          <Form.Field required>{PluginComponent}</Form.Field>
+        ) : (
+          <Form.Field>{PluginComponent}</Form.Field>
+        )}
```

Comment:
```suggestion
<Form.Field required={isRequired}>{PluginComponent}</Form.Field>
```
Bit simpler, right? ;)